postman-generator/src/main/scala/generator/DependantOperationResolver.scala

```diff
@@ -1,9 +1,9 @@
 package generator
 
 import io.apibuilder.spec.v0.models.{Operation, Resource, Service}
-import models.attributes.PostmanAttributes.ObjectReferenceAttrValue
-import models.attributes.PostmanAttributes
-import models.attributes.PostmanAttributes._
```

Comment:
can these models be deleted now?

Reply:
as for now, I deleted all entities from `PostmanAttributes`. 
However, there are still 2 little helper functions - that's why the file remains in the repo.